src/all/pandachaika/src/eu/kanade/tachiyomi/extension/all/pandachaika/PandaChaika.kt

```diff
@@ -42,6 +44,9 @@ class PandaChaika(
 
     private val json: Json by injectLazy()
 
+    private val fakkuRegex = Regex("""(?:https?://)?(?:www\.)?fakku\.net/manga/""")
+    private val ehentaiRegex = Regex("""(?:https?://)?e-hentai\.org/g/""")
```

Comment:
Would it make sense to also support exhentai here? or `www`?
```suggestion
    private val ehentaiRegex = Regex("""(?:https?://)?(?:www\.)?e[x-]hentai\.org/g/""")
```

Reply:
I have not seen any source URL coming from `exhentai`, or starting with `www`, but I'll add it still just to be safe. Thank you.